src/daemon/keymap.c

```diff
@@ -636,6 +638,23 @@ static inline void handle_bragi_key_input(unsigned char* kbinput, const unsigned
             SET_KEYBIT(kbinput, 104);   // voldn
             break;
         }
+    } else if(urbinput[0] == NKRO_MEDIA_IN && length == 5) {
```

Comment:
Having issues with getting this to work for the volume knob, maybe someone who is more familiar could help out.